integration_test/features/Fetch_JWT.feature

```diff
@@ -0,0 +1,26 @@
+@fetch @JWT
+Feature: Fetch JWT
+
+
+    @Sprint8 @SVID
+    Scenario: FJ_001 - Check that it is possible to fetch the JWT SVID
+        When I fetch "JWT" SVID
+        Then I check that the SVID is returned correctly
+
+
+    @Sprint8 @SVID
+    Scenario: FJ_002 - Check that it is possible to fetch JWT SVID with the server down
+        When The server is turned off
+        And  I fetch "JWT" SVID
+        Then I check that the SVID is returned correctly
```

Comment:
If the server is turned off it should return a SVID? 
I am not sure about that.